models/connect.go

```diff
@@ -0,0 +1,38 @@
+package models
+
+import (
+	"log"
+	"os"
+
+	"github.com/go-sql-driver/mysql"
+	"github.com/jmoiron/sqlx"
+)
+
+func Connect() (*sqlx.DB, error) {
+	conf := mysql.Config{
+		User:                 os.Getenv("MYSQL_USER"),
+		Passwd:               os.Getenv("MYSQL_PASSWORD"),
+		Net:                  "tcp",
+		Addr:                 os.Getenv("MYSQL_HOSTNAME") + ":" + os.Getenv("MYSQL_PORT"),
+		DBName:               "h24s17",
+		AllowNativePasswords: true,
+	}
+
+	db, err := sqlx.Open("mysql", conf.FormatDSN())
+	if err != nil {
+		log.Println("failed to open DB connection")
+
+		return db, err
+	}
+
+	err = db.Ping()
+	if err != nil {
+		log.Println("ping failed")
+
+		return db, err
+	}
+
+	log.Println("ping succeeded")
```

Comment:
DBのpingであることがわかると嬉しいかも
```suggestion
	log.Println("DB ping succeeded")
```